service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/validation/ItkValidatorTest.java

```diff
@@ -68,51 +86,51 @@ public void setUp() {
     private Element prepareItkPayloadsElement() {
         itkPayloads = mock(Element.class);
         itkPayloadsCount = mock(Attribute.class);
-        when(itkPayloadsCount.getValue()).thenReturn("1");
-        when(itkPayloads.attribute("count")).thenReturn(itkPayloadsCount);
+        lenient().when(itkPayloadsCount.getValue()).thenReturn("1");
```

Comment:
I'd avoid lenient at all - this test class might need some refactoring to handle new cases